pkg/tsdb/azuremonitor/azure-log-analytics-datasource.go

```diff
@@ -38,29 +39,26 @@ type AzureLogAnalyticsQuery struct {
 	RefID        string
 	ResultFormat string
 	URL          string
-	Model        *simplejson.Json
+	JSON         json.RawMessage
 	Params       url.Values
 	Target       string
+	TimeRange    backend.TimeRange
 }
 
 // executeTimeSeriesQuery does the following:
 // 1. build the AzureMonitor url and querystring for each query
 // 2. executes each query by calling the Azure Monitor API
 // 3. parses the responses for each query into the timeseries format
```

Comment:
```suggestion
// 3. parses the responses for each query into data frames
```
This comment was outdated even before you started working on this :)